data/cadastre/plss/index.html

```diff
@@ -144,16 +152,14 @@ <h5>Related Resources</h5>
 </div>
 <div class="grid package">
   <div class="grid__col grid__col--12-of-12">
-    <h3 id="PLSSPoints">PLSS Points AGRC</h3>
+    <h3 id="PLSSPoints">PLSS Point AGRC</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
-    <img class="inline-text-left outline" src="{% link images/plsspoint_agrc.jpg %}" alt="PLSS Points AGRC" />
-    <p><strong>The new release The Points AGRC data layer is under development. It is going to be changing as we move into the PLSS Fabric, as of January 2019. The downloads version is not from the current CadNSDI GIS Version 2.2 but the original PLSSPoints_AGRC. The latest submitted Tie Sheets (through June. 1, 2018) are included but the locations may not reflect adjusted locations from Fabric data.</strong></p>
-    <p>PLSSPoint_AGRC is a point dataset that represents the AGRC schema of the GCDB point layer. We are currently (Oct. 2019) redesigning it to reflect Points Submitted, Points incorporated into the fabric (Control), Points that have older Tie Sheets that are monument records with no X,Y coordinates. It also has a link to a copy of the tie sheet or the county website for those. The layer PLSSPoint_GCDB on the SGID and available for download, is the original source for this layer. This layer is used in the PLSS Corner Management App.</p>
-    <h5>Notes for PLSSPoint_AGRC feature class</h5>
+    <img class="inline-text-left outline" src="{% link images/plsspoint_agrc.jpg %}" alt="PLSS Point AGRC" />
+    <p><strong>The new release of Points_AGRC data layer is now created from the PLSSPoint_GCDB and a “Control” points feature class in the Fabric. The “Control” points are collected tie sheets points submitted to the AGRC that have been used to adjust and improve the PLSS Fabric.</h5>
     <ul>
```

Comment:
This ul should be removed

Reply:
sorry about the mess, go ahead and remove it. Thanks Not sure how I can edit it after I do a pull.